src/codeu/chat/util/Tokenizer.java

```diff
@@ -0,0 +1,68 @@
+
+package codeu.chat.util;
+
+import java.io.IOException;
+
+
+public final class Tokenizer{
+  private StringBuilder token;
+  private String source;
+  private int at;
+  
+  
+  public Tokenizer (String source){
+  }
+  public String next() throws IOException{
+    while(remaining() > 0 && Character.isWhitespace(peek())){
+      read();
+    }
+    if(remaining() <= 0){
+      return null;
+    }else if(peek() == '"'){
```

Comment:
To keep consistency with the rest of the code base, add spaces between () and {} except for function calls and definitions. 

For example this line would become `} else if (peek() == '"') {`